src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -47,13 +48,17 @@ pub struct GroupDeltasSummary {
     pub insert_sub_level_id: u64,
     pub insert_table_infos: Vec<SstableInfo>,
     pub group_construct: Option<GroupConstruct>,
-    pub group_destroy: Option<GroupDestroy>,
+    pub group_destroy: Option<CompactionGroupId>,
     pub group_meta_changes: Vec<GroupMetaChange>,
     pub group_table_change: Option<GroupTableChange>,
     pub new_vnode_partition_count: u32,
+    pub group_merge: Option<GroupMerge>,
 }
 
-pub fn summarize_group_deltas(group_deltas: &GroupDeltas) -> GroupDeltasSummary {
+pub fn summarize_group_deltas(
+    group_deltas: &GroupDeltas,
+    default_group_id: CompactionGroupId,
```

Comment:
what does the `default_group_id` mean?

Reply:
It represents the group_id to which the delta version belongs, just for group destory